Building package sentencepiece

PS C:\Users\abdul\vcpkg> ./vcpkg install sentencepiece
Computing installation plan...
The following packages will be built and installed:
    sentencepiece[core]:x86-windows -> 0.1.96
Detecting compiler hash for triplet x86-windows...
Restored 0 packages from C:\Users\abdul\AppData\Local\vcpkg\archives in 443.4 us. Use --debug to see more details.
Starting package 1/1: sentencepiece:x86-windows
Building package sentencepiece[core]:x86-windows...
-- Note: sentencepiece only supports static library linkage. Building static library.
CMake Error at scripts/cmake/vcpkg_check_linkage.cmake:70 (message):
  sentencepiece only supports static crt linkage
Call Stack (most recent call first):
  ports/sentencepiece/portfile.cmake:2 (vcpkg_check_linkage)
  scripts/ports.cmake:145 (include)


Error: Building package sentencepiece:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `git pull` and `.\vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+sentencepiece
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[sentencepiece]+Build+error
including:
  package: sentencepiece[core]:x86-windows -> 0.1.96
    vcpkg-tool version: 2022-02-24-7efe7f2b04912a7d674f86a406f9f2711c0221c6
    vcpkg-scripts version: 5962cf5a3 2022-03-07 (25 hours ago)

Additionally, attach any relevant sections from the log files above.

--- a/ports/sentencepiece/vcpkg.json
+++ b/ports/sentencepiece/vcpkg.json
@@ -3,7 +3,7 @@
   "version": "0.1.96",
   "port-version": 1,
   "description": "SentencePiece is an unsupervised text tokenizer and detokenizer mainly for Neural Network-based text generation systems where the vocabulary size is predetermined prior to the neural model training",
-  "supports": "!((windows | uwp) & !static))",
+  "supports": "!((windows | uwp) & !static)",
   "dependencies": [
     {
       "name": "vcpkg-cmake",

--- a/ports/sentencepiece/vcpkg.json
+++ b/ports/sentencepiece/vcpkg.json
@@ -1,7 +1,9 @@
 {
   "name": "sentencepiece",
   "version": "0.1.96",
+  "port-version": 1,
   "description": "SentencePiece is an unsupervised text tokenizer and detokenizer mainly for Neural Network-based text generation systems where the vocabulary size is predetermined prior to the neural model training",
+  "supports": "!((windows | uwp) & !static))",
   "dependencies": [
     {
       "name": "vcpkg-cmake",

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -1167,11 +1167,6 @@ sdl2-net:x64-uwp=fail
 # https://github.com/microsoft/vcpkg/issues/10918
 seal:arm-uwp=fail
 seal:x64-uwp=fail
-sentencepiece:arm64-windows=fail
-sentencepiece:arm-uwp=fail
-sentencepiece:x64-uwp=fail
-sentencepiece:x64-windows=fail
-sentencepiece:x86-windows=fail
 septag-sx:arm64-windows=fail
 septag-sx:arm-uwp=fail
 septag-sx:x64-uwp=fail

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6338,7 +6338,7 @@
     },
     "sentencepiece": {
       "baseline": "0.1.96",
-      "port-version": 0
+      "port-version": 1
     },
     "sentry-native": {
       "baseline": "0.4.15",

--- a/versions/s-/sentencepiece.json
+++ b/versions/s-/sentencepiece.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "a49c37df8b6b5a30cadbfd32b69b876185bfcaed",
+      "git-tree": "b243e85760f8f59100632997fdc34cdae47bcfd4",
       "version": "0.1.96",
       "port-version": 1
     },

--- a/versions/s-/sentencepiece.json
+++ b/versions/s-/sentencepiece.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "a49c37df8b6b5a30cadbfd32b69b876185bfcaed",
+      "version": "0.1.96",
+      "port-version": 1
+    },
+    {
       "git-tree": "b83e7b28dc28e405cfee3c94385e2039c8564040",
       "version": "0.1.96",
       "port-version": 0

